### ISSUE TITLE
Suggestion - settings backup

### ISSUE THREAD

**Issue by adskimomo** [Panthere/GoBot#87]
Allow export to file of settings along with import of settings from backup.

**Comment by Panthere** [comment]
Yes, I am planning on adding an import and export function and commandline support to instantly load another config file making it easier to open multiple instances :)

**Comment by ValleyInforge** [comment]
Panthere hi! 
I want donate you 10$ with bitcoin but first I need to ask you a quick question in private, you have telegram or mail?
